src/main/java/at/hannibal2/skyhanni/config/features/garden/PlotMenuHighlightingConfig.java

```diff
@@ -0,0 +1,45 @@
+package at.hannibal2.skyhanni.config.features.garden;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import at.hannibal2.skyhanni.utils.LorenzColor;
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorDraggableList;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PlotMenuHighlightingConfig {
+    @Expose
+    @ConfigOption(name = "Enabled", desc = "Highlights plots based on their status.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean enabled = false;
+
+    @Expose
+    @ConfigOption(name = "Statuses", desc = "Change the hierarchy of the different plot statuses.")
+    @ConfigEditorDraggableList
+    public List<PlotStatusType> deskPlotStatusTypes = new ArrayList<>();
```

Comment:
Maybe fill this list with the four values in preferred default order. (we still have the enabled toggle so its not default enabled)